.gitignore

```diff
@@ -17,6 +17,7 @@ pids
 
 dist/*
 !dist/index.html
+!dist/main.css
```

Comment:
Interesting how you did it this way. What would be the file(s) ignored if you exclude index.html and main.css?

Reply:
If I had any Javascript here, it would be the bundle that gets created.

Right now I don't have any client-side Javascript compiled for the proxy.